doc/metadata_template.md

```diff
@@ -38,4 +39,16 @@ for (MetadataTemplate templateInfo : templates) {
 
 [get-enterprise-metadata-1]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/MetadataTemplate.html#getEnterpriseMetadataTemplates(com.box.sdk.BoxAPIConnection,%20java.lang.String...)
 [get-enterprise-metadata-2]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/MetadataTemplate.html#getEnterpriseMetadataTemplates(java.lang.String,%20com.box.sdk.BoxAPIConnection,%20java.lang.String...)
-[get-enterprise-metadata-3]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/MetadataTemplate.html#getEnterpriseMetadataTemplates(java.lang.String,%20int,%20com.box.sdk.BoxAPIConnection,%20java.lang.String...)
\ No newline at end of file
+[get-enterprise-metadata-3]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/MetadataTemplate.html#getEnterpriseMetadataTemplates(java.lang.String,%20int,%20com.box.sdk.BoxAPIConnection,%20java.lang.String...)
+
+Delete a Metadata Template
+--------------------------
+
+The ['deleteMetadataTemplate(BoxAPIConnection, String, String)'][delete-metadata-template] method will remove a metadata template schema
```

Comment:
Can we include the parameter names in the method signature here?  It would probably be helpful for devs to know what those arguments mean.